### DISCUSSION TITLE
I want Use app for connect to ssh proxy

### DISCUSSION THREAD

**Discussion by AmirRezaie** [Eugeny/tabby#8421] [Q&A]
hi ... i have a german server .... with tunneling and napsternetV application on Iphone, I can connect to the VPN (ssh proxy) .... but in my mac, I can't connect to VPN ... can you help me?

**Comment by Mamooshi** [comment]
first change your mac network setting (enable proxy and port to any number like 6666)
then change your ssh port of server to any number like  9999
then in your terminal type:
ssh -D 127.0.0.1:9999 -N root@51.178.158.218 -p 9999
put your password and open your Chrome ... enjoy
for Firefox you have to enable proxy in setting as well as your system

**Comment by Mamooshi** [comment]
برای مک خیای راحته اول توی نتورکت قسمت پراکسی برو فعالش کن به ۱۲۷.۰.۰.۱ و یک پورت مثلا ۶۶۶۶
بعد که این دستورو توی ترمینال میزنی ازت پسورد سرورتو میخواد وقتی میزنی اتفاقی نمیفته همون لحظه باز میشه همه چیز برات
ssh -D 127.0.0.1:6666 -N root@ip-server -p (porte serveret)
همیشه حتما سعی کنید همون اول پورت اس اس اچ رو تغییر بدید روی ۲۲ فریز میشه دائم
